### PR TITLE
List dependencies

### PR DESCRIPTION
**Pull request by emya**
[#149936132]

### CODE REVIEW

File: lib/etl/job/manager.rb [hunk 1]
```diff
@@ -2,14 +2,33 @@
...
+      def ==(another_sock)
+        self.id == another_sock.id
+      end
+    end
+
     include Singleton
 
     # Map of id => Class
-    attr_accessor :job_classes, :job_class_factories
+    attr_accessor :job_classes, :job_class_factories, :job_parents
```
Comment (ChrisPRobinson) [inline lib/etl/job/manager.rb]: Do we need to expose job_parents anymore?

File: lib/etl/job/manager.rb [hunk 2]
```diff
@@ -33,6 +52,47 @@ def register(id, klass, klass_factory=nil)
...
+        pnode = @job_dependencies[pid_str]
+        pnode.add_child(node)
+      end
+
+      # Build a hash to keep dependencies
+      @job_dependencies[id_str] = node 
+      @job_dependencies[pid_str] = pnode 
+    end
+
+    def bfs
```
Comment (ChrisPRobinson) [inline lib/etl/job/manager.rb]: Can we rename to something like sorted_dependent_jobs?

File: lib/etl/job/manager.rb (hunk 1 above)
Comment (emya) [inline lib/etl/job/manager.rb]: No! I gonna get rid of it.

File: lib/etl/job/manager.rb (hunk 2 above)
Comment (emya) [inline lib/etl/job/manager.rb]: OK

**Pull request merged**